Description: Install things in the correct place and don't use sudo
Author: Iain R. Learmonth <irl@fsfe.org>
Last-Update: 2014-09-08

--- direwolf.orig/Makefile.linux
+++ direwolf/Makefile.linux
@@ -144,33 +144,37 @@
 	$(CC) $(CFLAGS) -c -o $@ $^
 
 
-# Optional installation into /usr/local/...
+# Optional installation into /usr/...
 # Needs to be run as root or with sudo.
 
 # TODO: Review file locations.
 
 install : direwolf decode_aprs tocalls.txt symbols-new.txt symbolsX.txt dw-icon.png direwolf.desktop
-	install direwolf /usr/local/bin
-	install decode_aprs /usr/local/bin
-	install text2tt /usr/local/bin
-	install tt2text /usr/local/bin
-	install ll2utm /usr/local/bin
-	install utm2ll /usr/local/bin
-	install aclients /usr/local/bin
-	install log2gpx /usr/local/bin
-	install -D --mode=644 tocalls.txt /usr/share/direwolf/tocalls.txt
-	install -D --mode=644 symbols-new.txt /usr/share/direwolf/symbols-new.txt
-	install -D --mode=644 symbolsX.txt /usr/share/direwolf/symbolsX.txt
-	install -D --mode=644 dw-icon.png /usr/share/direwolf/dw-icon.png
-	install -D --mode=644 direwolf.desktop /usr/share/applications/direwolf.desktop
-	install -D --mode=644 CHANGES.txt /usr/local/share/doc/direwolf/CHANGES.txt
-	install -D --mode=644 LICENSE-dire-wolf.txt /usr/local/share/doc/direwolf/LICENSE-dire-wolf.txt
-	install -D --mode=644 LICENSE-other.txt /usr/local/share/doc/direwolf/LICENSE-other.txt
-	install -D --mode=644 User-Guide.pdf /usr/local/share/doc/direwolf/User-Guide.pdf
-	install -D --mode=644 Raspberry-Pi-APRS.pdf /usr/local/share/doc/direwolf/Raspberry-Pi-APRS.pdf
-	install -D --mode=644 Raspberry-Pi-APRS-Tracker.pdf /usr/local/share/doc/direwolf/Raspberry-Pi-APRS-Tracker.pdf
-	install -D --mode=644 APRStt-Implementation-Notes.pdf /usr/local/share/doc/direwolf/APRStt-Implementation-Notes.pdf
-	install -D --mode=644 Quick-Start-Guide-Windows.pdf /usr/local/share/doc/direwolf/Quick-Start-Guide-Windows.pdf
+	mkdir -p debian/direwolf/usr/bin
+	mkdir -p debian/direwolf/usr/share/direwolf
+	mkdir -p debian/direwolf/usr/share/doc/direwolf
+	mkdir -p debian/direwolf/usr/share/applications
+	install direwolf debian/direwolf/usr/bin
+	install decode_aprs debian/direwolf/usr/bin
+	install text2tt debian/direwolf/usr/bin
+	install tt2text debian/direwolf/usr/bin
+	#install ll2utm debian/direwolf/usr/bin
+	#install utm2ll debian/direwolf/usr/bin
+	install aclients debian/direwolf/usr/bin
+	install log2gpx debian/direwolf/usr/bin
+	install -D --mode=644 tocalls.txt debian/direwolf/usr/share/direwolf/tocalls.txt
+	install -D --mode=644 symbols-new.txt debian/direwolf/usr/share/direwolf/symbols-new.txt
+	install -D --mode=644 symbolsX.txt debian/direwolf/usr/share/direwolf/symbolsX.txt
+	install -D --mode=644 dw-icon.png debian/direwolf/usr/share/direwolf/dw-icon.png
+	install -D --mode=644 direwolf.desktop debian/direwolf/usr/share/applications/direwolf.desktop
+	install -D --mode=644 CHANGES.txt debian/direwolf/usr/share/doc/direwolf/CHANGES.txt
+	install -D --mode=644 LICENSE-dire-wolf.txt debian/direwolf/usr/share/doc/direwolf/LICENSE-dire-wolf.txt
+	install -D --mode=644 LICENSE-other.txt debian/direwolf/usr/share/doc/direwolf/LICENSE-other.txt
+	install -D --mode=644 User-Guide.pdf debian/direwolf-docs/usr/share/doc/direwolf/User-Guide.pdf
+	install -D --mode=644 Raspberry-Pi-APRS.pdf debian/direwolf-docs/usr/share/doc/direwolf/Raspberry-Pi-APRS.pdf
+	install -D --mode=644 Raspberry-Pi-APRS-Tracker.pdf debian/direwolf-docs/usr/share/doc/direwolf/Raspberry-Pi-APRS-Tracker.pdf
+	install -D --mode=644 APRStt-Implementation-Notes.pdf debian/direwolf-docs/usr/share/doc/direwolf/APRStt-Implementation-Notes.pdf
+	install -D --mode=644 Quick-Start-Guide-Windows.pdf debian/direwolf-docs/usr/share/doc/direwolf/Quick-Start-Guide-Windows.pdf
 
 
 # The Raspberry Pi has ~/Desktop but Ubuntu does not.
